l10n_jp_post_code_propose_partner_address/models/res_partner.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import jaconv
+import json
+import urllib
+
+from odoo import models, fields, api, _
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    country_id = fields.Many2one(
+        default=lambda self: self.env.ref('base.jp')
+    )
```

Comment:
I don't fancy the design of fixing the default value to Japan here. We may include the country proposal in the postal-code based address proposal.